compiler/control/OMROptions.cpp

```diff
@@ -4792,6 +4843,7 @@ OMR::Options::setVerboseBits(char *option, void *base, TR::OptionTable *entry)
 char *
 OMR::Options::setVerboseBitsHelper(char *option, VerboseOptionFlagArray *verboseOptionFlags, uintptr_t defaultVerboseFlags)
    {
+   TR_VerboseLog::vlogAcquire();
```

Comment:
Another place where we are acquiring the lock with the possibility of never actually doing a write.